pkg/ssh/hostkey/callbacks.go

```diff
@@ -132,14 +132,14 @@ func ensureDir(path string) error {
 	return nil
 }
 
-func ensureFile(path string) error {
-	if fileExists(path) {
+func ensureFile(filePath string) error {
+	if fileExists(filePath) {
 		return nil
 	}
-	if err := ensureDir(filepath.Dir(path)); err != nil {
+	if err := ensureDir(path.Dir(filePath)); err != nil {
```

Comment:
These are local paths so `filepath` is ok here.

Reply:
Oh, yes, got carried away with the change :)